images/manipulation.go

```diff
@@ -155,14 +155,42 @@ func EncodePNG(img *image.RGBA) ([]byte, error) {
 	return resultImg.Bytes(), nil
 }
 
-func CreateCircle(img image.Image) *image.RGBA {
+// helper function to calculate the minimum of two integers
```

Comment:
The convention I think is to start the documentation with the name of the function.

Reply:
Got rid of it, I thought about it a bit more and it seems like that operation was not necessary.